sdk/storage/storage-file-share/src/generated/src/models/index.ts

```diff
@@ -18,13 +17,13 @@ export interface AccessPolicy {
    * **NOTE: This entity will be treated as a string instead of a Date because the API can
    * potentially deal with a higher precision value than what is supported by JavaScript.**
    */
-  start?: string;
+  startsOn?: string;
```

Comment:
Please do not update the `generated` folder manually.
Swagger transformations have to be added and the code has to be generated again.

Reply:
Agree